test/test_sample.py

```diff
@@ -4,6 +4,9 @@
 
 import sys
 from Code.main import increment, decrement, square, divide
+from Code.sym import Sym
+
```

Comment:
Please change file name to test_eg.py and name of tests to for example: test_eg_sym()

Also, are we allowed to use pytest? Bcoz in lua he has written his own test suite and printed failures and everything manually using print statements. If we use pytest, our final output on terminal might deviate slightly from the lua output. I think for now we can submit HW2 with pytest and ask in discord on tuesday. If they tell us to make it from scratch, we can resubmit HW2 later

Reply:
As discussed, this will be taken up on a different thread.